src/main/webapp/app/exercise/exercise-scores/exercise-scores.component.ts

```
@@ -220,10 +228,8 @@ export class ExerciseScoresComponent implements OnInit, OnDestroy {
             results?.sort((result1, result2) => (result1.id ?? 0) - (result2.id ?? 0));
             const resultsWithoutAthena = results?.filter((result) => result.assessmentType !== AssessmentType.AUTOMATIC_ATHENA);
             if (resultsWithoutAthena?.length != 0) {
-                if (resultsWithoutAthena?.[0].submission) {
-                    participation.submissions = [resultsWithoutAthena?.[0].submission];
-                } else if (results?.[0].submission) {
-                    participation.submissions = [results?.[0].submission];
+                if (participation && participation.submissions && participation?.submissions[0]) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)